CONTRIBUTING.md

```
@@ -156,9 +158,10 @@ and set the values of the variables according to your environment.
 >     browser testing, but remote development and production deployments need
 >     HTTPS for browser Push API subscriptions.
 >
->  -  `KV_URL` may point to a file when running only `mise run dev:graphql`.
->     Redis is required whenever the worker is running and for every production
->     process, because those processes must share one coherent KV store.
+>  -  `KV_URL` may point to a file when running only `mise run dev:graphql` or
```

markdownlint-cli2 (0.23.0)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)